content-src/components/TopSites/TopSiteForm.jsx

```diff
@@ -205,25 +203,23 @@ export class TopSiteForm extends React.PureComponent {
       <form className="topsite-form" onSubmit={onSubmit}>
         <div className="form-input-container">
           <h3 className="section-title">
-            <FormattedMessage id={showAsAdd ? "topsites_form_add_header" : "topsites_form_edit_header"} />
+            <span data-l10n-id={showAsAdd ? "newtab_topsites_add_header" : "newtab_topsites_edit_header"} />
```

Comment:
We shouldn't need additional `<span>` children for text parents, so the `data-l10n-id` should be able to go directly on the `<h3>` here

Reply:
Normally true but it would remove the css selector `.collapsible-section .section-title span` ([here](https://searchfox.org/mozilla-central/source/browser/components/newtab/css/activity-stream-windows.css#1577)) that makes this gray instead of black so I chose to keep it here to preserve that.

Reply:
@Mardak where ever possible may be we should updates css to stop using span